jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/menu/ManagePerspectivesSelector.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.menu;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import org.jboss.errai.common.client.api.elemental2.IsElement;
+import org.jboss.errai.common.client.dom.EventListener;
+import org.jboss.errai.common.client.ui.ElementWrapperWidget;
+import org.jboss.errai.security.shared.api.identity.User;
+import org.jbpm.workbench.common.client.PerspectiveIds;
+import org.jbpm.workbench.common.client.resources.i18n.Constants;
+import org.uberfire.client.mvp.ActivityManager;
+import org.uberfire.client.mvp.PlaceManager;
+
+import org.uberfire.client.workbench.events.PerspectiveChange;
+import org.uberfire.mvp.PlaceRequest;
+import org.uberfire.mvp.impl.DefaultPlaceRequest;
+import org.uberfire.security.ResourceRef;
+import org.uberfire.security.authz.AuthorizationManager;
+import org.uberfire.workbench.model.ActivityResourceType;
+
+@ApplicationScoped
+public class ManagePerspectivesSelector {
```

Comment:
Because this class is used in presenter classes, it would be useful to make it an interface so it makes easier to mock when needed. 
Also perhaps name it _ManageSelector_ and _ManagerSelectorImpl_ to match with the view classes?